src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -54,7 +54,7 @@ def _get_default_install_location(exe_name):
 register_cli_argument('acs', 'orchestrator_type', **enum_choice_list(ContainerServiceOchestratorTypes))
 #some admin names are prohibited in acs, such as root, admin, etc. Because we have no control on the orchestrators, so default to a safe name.
 register_cli_argument('acs', 'admin_username', options_list=('--admin-username',), default='azureuser', required=False)
-register_cli_argument('acs', 'dns_name_prefix', options_list=('--dns-prefix', '-d'))
+register_cli_argument('acs', 'dns_name_prefix', options_list=('--dns-prefix', '-d'), default=None, required=False)
```

Comment:
Please remove `default=None, required=False` as there's no need to specify it here.
Instead, in the method definition for `acs_create()`, make `dns_name_prefix` have a default of `None` i.e. `dns_name_prefix=None`.
The CLI will automatically pick up that it's an optional parameter that way.